images/toolbox/Dockerfile

```diff
@@ -35,8 +55,15 @@ RUN curl -Ssf "https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m).zip" \
     chmod +x "${awscli_install_dir}/aws/dist/aws"
 ENV PATH=$PATH:$awscli_install_dir/aws/dist
 
+ARG wget2_tarball
+COPY --from=builder /tmp/$wget2_tarball /tmp/wget
+WORKDIR /tmp/wget
+RUN make install; \
+    rm -rf /tmp/wget
```

Comment:
I wasn't entirely sure off the top of my head whether this would work (i.e. unlinking the current working directory). Spoiler alert: it does!